.github/workflows/sonarcloud.yml

```diff
@@ -75,18 +75,24 @@ jobs:
 
       - name: Initialize CodeQL
         if: steps.changed-files-yaml.outputs.backend_any_changed == 'true' || steps.branch-name.outputs.is_default == 'true'
-        uses: github/codeql-action/init@65c74964a9ed8c44ed9f19d4bbc5757a6a8e9ab9
+        uses: github/codeql-action/init@v3
         with:
           languages: ${{ matrix.language }}
 
       - name: Build Prime Router Package
-        if: steps.changed-files-yaml.outputs.backend_any_changed == 'true' || steps.branch-name.outputs.is_default == 'true'
```

Comment:
@snesm Did you mean to remove the `if` condition?

Reply:
Good catch.  I went through a lot of iterations of this because the backend build test suite had a flaw, and this got dropped.